### ISSUE TITLE
Next page missing from searx v. 0.14.0 (only one page of results)

### ISSUE THREAD

**Issue by beatstick** [searx/searx#1404]
I only ever get one page of results since the update to 0.14.0 on my self-hosted searx instance. 
Is this a bug or am I missing some obvious setting? I haven't found anything in the config file.

**Comment by Pofilo** [comment]
It depends of your configuration. If you go into the **Preferences** in the plugin section, you have one name **Infinite scroll**. Is it activated ?

**Comment by beatstick** [comment]
I tried activating infinite scroll as well. I still only get one page of results. It makes no difference.

**Comment by jordemort** [comment]
Seeing the same problem on my instance (0.14, just refreshed my container from master today)

**Comment by Perflyst** [comment]
I still have the same issue with v0.16.0 and v0.17.0

**Comment by unixfox** [comment]
Please paste the enabled engines in the preferences and if it's possible the URL to the Searx instance used.

Usually this bug is related to an engine that silently fails because the server is rate limited or blocked by the service used.

**Comment by unixfox** [comment]
Probably duplicate of https://github.com/searx/searx/issues/2053
